org.opent2t.sample.hub.superpopular/com.insteon.hub/js/thingTranslator.js

```diff
@@ -96,22 +96,24 @@ class Translator {
                                                     }
                                                     break;
                                             }
-                                        } else {
-                                            console.log(deviceStatus);  //failed to get device status
                                         }
                                         
                                         // Create a translator for this device and get the platform information, possibly expanded
                                         return OpenT2T.createTranslatorAsync(opent2tInfo.translator, { 'deviceInfo': deviceInfo, 'hub': this })
                                             .then((translator) => {
-
                                                 // Use get to translate the Insteon formatted device that we already got in the previous request.
                                                 // We already have this data, so no need to make an unnecesary request over the wire.
                                                 return OpenT2T.invokeMethodAsync(translator, opent2tInfo.schema, 'get', [expand, deviceData])
                                                     .then((platformResponse) => {
                                                         return platformResponse;
                                                     });
+                                            }).catch((err) => {
+                                                console.log('warning: OpenT2T.createTranslatorAsync error - ' + err);
+                                                return Promise.resolve(undefined);
                                             });
                                         
+                                    }).catch((err) => { 
+                                        throw err;
                                     });
```

Comment:
I'm not sure I follow the flow here.  If _getCommandResponse rejects, or throws, it will be caught here and re-thrown.  Does the uncaught exception get converted into a rejection?  We'd prefer to always explicitly reject if possible inside of asynchronous tasks.  Rejections will get converted into OpenT2TError classes by OpenT2T.